src/imp/openssl.rs

```diff
@@ -263,7 +263,9 @@ impl TlsConnector {
         supported_protocols(builder.min_protocol, builder.max_protocol, &mut connector)?;
 
         for cert in &builder.root_certificates {
-            connector.cert_store_mut().add_cert((cert.0).0.clone())?;
+            if let Err(err) = connector.cert_store_mut().add_cert((cert.0).0.clone()) {
```

Comment:
I feel like we should do better here and check for the duplicate cert error in particular, but I can do that in a follow-up.

Reply:
Yep!

In practice it's not that big of a problem since I think the only other error that can come out of this method is an allocation failure but it still seems more explicit at least.